jetty-ee11/pom.xml

```diff
@@ -236,379 +236,4 @@
-                      <rule>
-                        <!-- for ee11, we need to stay on 3.1.x of ws.rs.api spec -->
-                        <groupId>jakarta.ws.rs</groupId>
-                        <artifactId>jakarta.ws.rs-api</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!3.1).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- for ee11, we need to stay on 4.0.x of ws.rs.api impl -->
-                        <groupId>com.sun.xml.ws</groupId>
-                        <artifactId>jaxws-rt</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!4.0).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- for ee11, we need to stay on 4.0.x of xml.bind impl -->
-                        <groupId>org.glassfish.jaxb</groupId>
-                        <artifactId>jaxb-runtime</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!4.0.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- for ee11, we need to stay on 4.0.x of xml.bind-api -->
-                        <groupId>jakarta.xml.bind</groupId>
-                        <artifactId>jakarta.xml.bind-api</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!4.0.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                      <rule>
-                        <!-- for ee11, we need to stay on 4.0.x of xml.ws-api -->
-                        <groupId>jakarta.xml.ws</groupId>
-                        <artifactId>jakarta.xml.ws-api</artifactId>
-                        <ignoreVersions>
-                          <ignoreVersion>
-                            <type>regex</type>
-                            <version>^(?!4.0.).+</version>
-                          </ignoreVersion>
-                        </ignoreVersions>
-                      </rule>
-                    </rules>
-                  </ruleSet>
-                </configuration>
-              </execution>
-            </executions>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
-  </profiles>
```

Comment:
Lets hold off on cleaning up the pom right now.
We can remove this section later, once we've confirmed the new dependabot configs.